kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -335,9 +335,9 @@
       markAsComplete() {
         this.wasIncomplete = false;
       },
-      genContentLink(id, isLeaf) {
+      genContentLink(id, kind) {
         return {
-          name: isLeaf ? PageNames.TOPICS_CONTENT : PageNames.TOPICS_TOPIC,
```

Comment:
The `isLeaf` field is something we just added to the ContentNode API to avoid this `kind === TOPIC` check. Was it not working?

Reply:
I must have had a merge conflict and committed an older version of the code. I'll make sure it's fixed before the PR is finalized. Thanks for catching this!